fabric-textures-v0/src/main/java/net/fabricmc/fabric/api/client/texture/ColorSniffer.java

```diff
@@ -0,0 +1,52 @@
+package net.fabricmc.fabric.impl.client.texture;
+
+import net.fabricmc.fabric.mixin.client.texture.SpriteAccessor;
+import net.minecraft.client.texture.NativeImage;
+import net.minecraft.client.texture.Sprite;
+
+/**
+ * Utility class for averaging Sprite colors.
+ * You should call one of these methods!
+ */
+public class ColorSniffer {
+
+	/**
+	 * @param sprite The Sprite you want to get the average color from.
```

Comment:
Jd needs a body. If anything, move the stuff in return tag to the jd body like google java style suggests.